### PR TITLE
VR-5936 Add brief examples to Strategy, Resource, Metric, and Rule docstrings

### PR DESCRIPTION
**Pull request by nhatsmrt**
@convoliution See if the example makes sense.
Also fix some small mix-up in docstrings for autoscaling metrics, and docstring for strategy.

### CODE REVIEW

File: client/verta/verta/deployment/resources.py [hunk 1]
```diff
@@ -36,6 +36,13 @@ class CpuMillis(_Resource):
     """
     Number of CPU milli allowed for Endpoint. Must be an integer greater than 0.
 
+    Examples
+    --------
+    .. code-block:: python
+
+        from verta.deployment.resources import CpuMillis
+        cpu = CpuMillis(128)
```
Comment (convoliution) [inline client/verta/verta/deployment/resources.py]: 1000 millis = 1 CPU core, so a nice fraction of 1000 like `250` makes more sense than `128`.
```suggestion
        cpu = CpuMillis(250)
```

File: client/verta/verta/deployment/update/_strategies.py [hunk 2]
```diff
@@ -30,7 +42,19 @@ def _as_build_update_req_body(self, build_id):
             'strategy': self._STRATEGY,
         }
 
+
 class CanaryUpdateStrategy(_UpdateStrategy):
+    """
+    Represents canary update strategy for Endpoint.
+
```
Comment (convoliution) [inline client/verta/verta/deployment/update/_strategies.py:49]: Could you also do me a favor and move the **Parameters** from the init docstring into this class docstring (and then remove the init docstring)? See [`Client`](https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/client.py) for what that would look like.

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/deployment/update/_strategies.py (hunk 2 above)
Comment (nhatsmrt) [reply client/verta/verta/deployment/update/_strategies.py:49]: Done.

File: client/verta/verta/deployment/resources.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/deployment/resources.py]: Fixed.

**Review by convoliution — APPROVED** [review]

**Pull request merged**